### DISCUSSION TITLE
If I have the title and the author in the YAML metadata, is it expected that both will be visually exported in the pdf?

### DISCUSSION THREAD

**Discussion by aabdou** [jgm/pandoc#10443] [General]
I use Debian and installed all the pdf + yaml required packages for pandoc. I have a YAML block at the beginning of my markdown. I tried exporting it with the pdflatex into a PDF but both the author and title are in the pdf just above the content. Is this expected?

**Comment by badumont** [comment]
Yes, but you can change the template used to produce the intermediary LaTeX file (see the manual).
